front_end/models/react_native/ReactDevToolsBindingsModel.ts

```diff
@@ -47,11 +47,22 @@ export class ReactDevToolsBindingsModel extends SDK.SDKModel.SDKModel {
   private bindingCalled(event: BindingCalledEventTargetEvent): void {
     const serializedMessage = event.data.payload;
 
+    let parsedMessage = null;
+
     try {
-      const {domain, message} = JSON.parse(serializedMessage);
-      this.dispatchMessageToDomainEventListeners(domain, message);
+      parsedMessage = JSON.parse(serializedMessage);
     } catch (err) {
-      throw new Error('Failed to parse bindingCalled event payload:', err);
```

Comment:
`Failed to parse bindingCalled event payload` was raised even though there was an error coming from a throw in the event listener.
What's more, the error was incorrect.
`throw new Error('Failed to parse bindingCalled event payload:', err);` was swallowing the `err` because this is not a valid thing to pass to `new Error`

Reply:
Oh, probably a typo, it should be `throw new Error('Failed to parse bindingCalled event payload:', {cause: err});

Reply:
Why event listeners were throwing an error though?

Reply:
used `{cause: err}`